out_xml/01_spatial.xml

```diff
@@ -192,7 +192,7 @@
     <lineage>
       <procstep>
         <procdesc>At the core of the modeling framework is a deep learning model that uses inputs of XX.</procdesc>
```

Comment:
Still need description here.

Reply:
Yeah, there's a fair amount of info still to add. Planning to do it in response to Hayley's review since she has already provided that.